tests/components/deconz/test_fan.py

```diff
@@ -0,0 +1,185 @@
+"""deCONZ fan platform tests."""
+from copy import deepcopy
+
+from homeassistant.components import deconz
+from homeassistant.components.deconz.gateway import get_gateway_from_config_entry
+import homeassistant.components.fan as fan
+from homeassistant.const import STATE_OFF, STATE_ON
+from homeassistant.setup import async_setup_component
+
+from .test_gateway import DECONZ_WEB_REQUEST, setup_deconz_integration
+
+from tests.async_mock import patch
+
+FANS = {
+    "1": {
+        "etag": "432f3de28965052961a99e3c5494daf4",
+        "hascolor": False,
+        "manufacturername": "King Of Fans,  Inc.",
+        "modelid": "HDC52EastwindFan",
+        "name": "Ceiling fan",
+        "state": {
+            "alert": "none",
+            "bri": 254,
+            "on": False,
+            "reachable": True,
+            "speed": 4,
+        },
+        "swversion": "0000000F",
+        "type": "Fan",
+        "uniqueid": "00:22:a3:00:00:27:8b:81-01",
+    }
+}
+
+
+async def test_platform_manually_configured(hass):
+    """Test that we do not discover anything or try to set up a gateway."""
+    assert (
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6319846)